externs/shaka/player.js

```diff
@@ -950,6 +951,8 @@ shaka.extern.OfflineConfiguration;
  *   The preferred role to use for text tracks.
  * @property {number} preferredAudioChannelCount
  *   The preferred number of audio channels.
+ * @property {boolean} preferredTextLanguageForced
+ *   If true, a forced text track is preferred.  Defaults to false.
```

Comment:
I did not understand the meaning of this flag from the name alone.  Now that I am reading the description, I feel like "preferForcedSubs" would make more sense.

Also, can you please clarify the behavior in combination with other settings?  For example, it seems like this is to choose between multiple tracks with the same language and different "forced" settings.  So, if my content has en & en-forced, and I prefer en, this flag controls which one I get.  But if my content has en & fr & en-forced, and I prefer fr, this flag won't have any effect.  Is that correct?

Reply:
I have changed the name, I have put a more detailed description and I have added a new test to make everything clearer.